python/dgl/backend/backend.py

```diff
@@ -1125,6 +1125,38 @@ def replace_inf_with_zero(x):
     """
     pass
 
+def count_nonzero(input):
+    """Return the count of non-zero values in the tensor input.
+
+    Parameters
+    ----------
+    input : Tensor
+        The tensor to be counted
+
+    Returns
+    -------
+    Integer
+        The result 
+    """
+    pass
+
+def add(input, other):
```

Comment:
I think you can directly write `+` instead of defining this new function.

Reply:
Good suggestion. Updated.